reference/api/search.md

```diff
@@ -638,6 +642,10 @@ Though it is not necessary to use `highlightPreTag` and `highlightPostTag` in co
 
 Adds a `_matchesPosition` object to the search response that contains the location of each occurrence of queried terms across all fields. This is useful when you need more control than offered by our [built-in highlighting](#attributes-to-highlight). `showMatchesPosition` only works for strings, numbers, and arrays of strings and numbers.
 
+::: warning
+`showMatchesPosition` will consider all attributes that contain words matching the query terms even if they are not set as `searchableAttributes`.
```

Comment:
"consider" is a bit unclear here. Can we find a better verb, or group of words?

I'm contrasting it to the new warning on L#568, where "highlight" tells me what happens even if I haven't read anything else about that parameter.

Reply:
I can only think of "takes into account", but that's not helpful either 🤔

Reply:
What about "returns match location"?

```suggestion
`showMatchesPosition` returns match location for all attributes that contain words matching the query terms even if they are not set as `searchableAttributes`.
```